src/ontology/reports/sync-synonym/review-qc-duplicate-exact-synonym-no-abbrev.tsv

```diff
@@ -583,8 +583,8 @@ JMP syndrome	MONDO:0054698	OMIM:256040	updated
 Jacobs syndrome	MONDO:0008828	DOID:0090127	unconfirmed		
 Jacobs syndrome	MONDO:0008828	OMIM:208250	confirmed		
 Jacobs syndrome	MONDO:0019339	Orphanet:8	added		
-Jarcho-Levin syndrome	MONDO:0000359	DOID:0050568	updated		
-Jarcho-Levin syndrome	MONDO:0010180	Orphanet:2311	confirmed		
+Jarcho-Levin syndrome	MONDO:0000359	DOID:0050568	updated	http://purl.obolibrary.org/obo/mondo#DEPRECATED	
```

Comment:
I ran these outputs on develop and put them in #767 so that we could compare the outputs of #767 in this mini build to the current state of develop.

I'm surprised to see these synonym types added to this review file here. I thought they were there before. But it's good they're showing up now. I'll double check that these ones being added are indeed correct.